src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerMessage.cs

```diff
@@ -540,8 +585,24 @@ public LogValues(LogValuesFormatter formatter, T0 value0)
                 }
             }
 
+            public override string ToString()
+            {
+                Span<char> output = stackalloc char[1024];
```

Comment:
How do we know `1024` is big enough?

Reply:
This should use ValueStringBuilder

Reply:
It's not. This was just to please some initial tests and I'll need to amend it in a much smarter way 😄